add information about ICU normalisation (#570)
arangodb/docs: 3.8/aql/functions-string.md

String functions
================

> Please note that you may neet to execute [`NORMALIZE_UTF8()`](#normalize_utf8)
> on input strings if you intend to i.e. locate glyphs that have multiple 
> representations in Unicode / UTF-8.
> 

Reviewer: ```suggestion
```

CRC32("foobar") // "D5F5C7F"
```

NORMALIZE_UTF8
--------------

> `NORMALIZE_UTF8('some utf8 text')`

Reviewer: I think this signature does not match the rest of the AQL functions signatures in the docs.
It should rather be something like
```
NORMALIZE_UTF8(string) → normalized
```
along with a description of the input parameter and the return value. Basically like how it is for all other AQL functions.

In Unicode there may be more than one representation of a glyph. 

> Use [ICU Normalisation](http://www.unicode.org/reports/tr15/) in order to convert

Reviewer: We agreed on using AE consistently.
```suggestion
Use [ICU Normalization](http://www.unicode.org/reports/tr15/) in order to convert
```

> to a similar character. It may also be usefull to run this normalisation before 

Reviewer: ```suggestion
to a similar character. It may also be useful to run this normalization before 
```

> writing the documents into the database. If you have strings with different representation, 

Reviewer: ```suggestion
writing the documents into the database. If you have strings with different representations, 
```

CRC32("foobar") // "D5F5C7F"
```

> NORMALIZE_UTF8
> --------------
> 
> `NORMALIZE_UTF8(text) → normalized`
> 
> In Unicode there may be more than one representation of a glyph. 
> Use [ICU Normalization](http://www.unicode.org/reports/tr15/) in order to convert
> to a similar character. It may also be useful to run this normalization before 
> writing the documents into the database. If you have strings with different representations, 
> functions like [`FIND_FIRST`](#find_first) may not find all matches you desire.
> 
> - **text** (string): a UTF8-string
> - returns **nomalized UTF8-string** (string): the normalized string.
> 

Reviewer: ```suggestion
```
